core/src/main/java/com/pholser/junit/quickcheck/internal/generator/GeneratorRepository.java

```diff
@@ -59,6 +62,11 @@ a copy of this software and associated documentation files (the
 import static com.pholser.junit.quickcheck.internal.Reflection.*;
 
 public class GeneratorRepository implements Generators {
```

Comment:
General note: I tend to favor static-importing names when doing so doesn't introduce confusion. For example, here I'd static-import `Collections.unmodifiableSet`, but not `Stream.of`.